test/deploy-prepare-and-test-pipeline.sh

```diff
@@ -0,0 +1,157 @@
+    --source-account-name ${SOURCE_STORAGE_ACCOUNT_NAME} \
+    --source-account-key ${SOURCE_STORAGE_ACCOUNT_KEY} \
+    --source-blob config/config-aoi.json
+
+  az storage blob copy start \
+    --destination-blob config/config-img-convert-png.json \
+    --destination-container ${DESTINATION_CONTAINER_NAME} \
+    --account-name ${DESTINATION_STORAGE_ACCOUNT_NAME} \
+    --account-key ${STORAGE_ACCOUNT_KEY} \
+    --source-container ${SOURCE_CONTAINER_NAME} \
+    --source-account-name ${SOURCE_STORAGE_ACCOUNT_NAME} \
+    --source-account-key ${SOURCE_STORAGE_ACCOUNT_KEY} \
+    --source-blob config/config-img-convert-png.json
+
+  az storage blob copy start \
+    --destination-blob config/config-pool-geolocation.json \
+    --destination-container ${DESTINATION_CONTAINER_NAME} \
+    --account-name ${DESTINATION_STORAGE_ACCOUNT_NAME} \
+    --account-key ${STORAGE_ACCOUNT_KEY} \
+    --source-container ${SOURCE_CONTAINER_NAME} \
+    --source-account-name ${SOURCE_STORAGE_ACCOUNT_NAME} \
+    --source-account-key ${SOURCE_STORAGE_ACCOUNT_KEY} \
+    --source-blob config/config-pool-geolocation.json
+fi
+
+# Install Pipeline
+echo "Preparing commands to install pipeline components"
+mkdir /tmp/packaged-pipeline
+unzip ${PIPELINE_NAME}.zip -d /tmp/packaged-pipeline/
+echo "#!/usr/bin/env bash" > /tmp/install-pipeline.sh
+echo "set -ex" >> /tmp/install-pipeline.sh
+chmod +x /tmp/install-pipeline.sh
+python3 ${PRJ_ROOT}/test/create-pipeline-install-commands.py \
+    --file_name "/tmp/packaged-pipeline/pipeline/${PIPELINE_FILE_NAME}" \
+    --resource_group_name ${SYNAPSE_RESOURCE_GROUP_NAME} \
+    --workspace_name ${SYNAPSE_WORKSPACE_NAME} >> /tmp/install-pipeline.sh
+
+echo "Installing pipeline components"
+/tmp/install-pipeline.sh
+
+# Create pipeline parameter file
+if [[ "${SUBSTITUTE_PARAMS,,}" == "true" ]]; then
+    echo "Generating parameter file for pipeline execution"
+    BATCH_ACCOUNT_ID=$(az batch account list --query "[?name == '${BATCH_ACCOUNT_NAME}'].id" -o tsv)
+    BATCH_ACCOUNT_RG_NAME=$(az resource show --ids ${BATCH_ACCOUNT_ID} --query resourceGroup -o tsv)
+    BATCH_ACCOUNT_LOCATION=$(az batch account show  --name ${BATCH_ACCOUNT_NAME} --resource-group ${BATCH_ACCOUNT_RG_NAME} --query "location" --output tsv)
+
+    export RAW_DATA_STORAGE_ACCOUNT_CONTAINER_NAME=${DESTINATION_CONTAINER_NAME}
+    export RAW_DATA_STORAGE_ACCOUNT_NAME=${DESTINATION_STORAGE_ACCOUNT_NAME}
+    export RAW_DATA_STORAGE_ACCOUNT_KEY=${DESTINATION_STORAGE_ACCOUNT_RG_NAME}
+    export BATCH_ACCOUNT_NAME
+    export BATCH_JOB_NAME
+    export BATCH_ACCOUNT_LOCATION
+    export AOI=${AOI_PARAMETER_VALUE}
+
+    envsubst < ${PARAMETER_FILE_PATH} > /tmp/parameters.json
+    export PARAMETER_FILE_PATH="/tmp/parameters.json"
+fi
+
+# Execute pipeline
+echo "Executing ${PIPELINE_NAME} pipeline"
+PIPELINE_RUN_ID=$(az synapse pipeline create-run \
+    --workspace-name ${SYNAPSE_WORKSPACE_NAME} \
+    --name "E2E Custom Vision Model Flow" \
```

Comment:
Can we have the pipeline name passed to this script so that we can keep the script generic and use the same script for other pipelines?

Reply:
If not big effort, lets have it in this PR or next PR is fine too